Keas.Mvc/Keas.Mvc.csproj

```diff
@@ -23,6 +23,7 @@
     <PackageReference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="3.1.5" />
     <PackageReference Include="Microsoft.EntityFrameworkCore.Sqlite" Version="3.1.5" />
     <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="3.1.5" />
+    <PackageReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Design" Version="3.1.4" />

```

Comment:
This one snuck in again

Reply:
yea seems VS adds that automatically when you do certain scaffold-able options, like adding a view.  I double checked and it restores fine on mac and doesn't cause any issues so we might as well just keep it since it'll try to come back itself anyway